python/cugraph/cugraph/community/louvain.py

```diff
@@ -89,6 +93,14 @@ def louvain(G, max_level=None, max_iter=None, resolution=1.0, threshold=1e-7):
 
     """
 
+    isolated_vertices = list()
```

Comment:
Perhaps a FIXME indicating this is a temporary solution.

If we add a mechanism in the graph construction calls to support isolated vertices through the C API (the C++ interface already supports this) then there will be no need to compute isolated vertices here.

Reply:
Added